frame/support/test/tests/construct_runtime_ui/no_std_genesis_config.stderr

```diff
@@ -36,6 +36,28 @@ help: consider importing this enum
 1  | use frame_system::RawOrigin;
    |
 
+error[E0433]: failed to resolve: use of undeclared crate or module `system`
```

Comment:
This extra error likely appears because of a derived `TypeInfo` attempting to resolve a `system::Call` type.

I see how this ui test and others in `construct_runtime_ui` are just really comparing the first error message anyway, because all of them have this error of using `system` instead of `frame_system`.

Reply:
indeed this UI tests could be improved but nothing has worsen here